docker-compose.yaml

```diff
@@ -11,7 +11,7 @@ services:
     container_name: ankerctl_nginx
     restart: unless-stopped
     volumes:
-      - ./nginx.conf:/etc/nginx/nginx.conf
+      - ./templates/docker/nginx.conf:/etc/nginx/nginx.conf
```

Comment:
I'm all for moving things out of the root dir, but `templates/` is definitely not the way to go. That dir is exclusively for the `transwarp` templates, used to compile the files in `specification/` into rendered code files (python, for now).

How about we make a top-level `docker/` dir, and put everything docker-related there?